src/Rector/MethodCall/SimplifyFormRenderingRector.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Type\ObjectType;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Symfony\Tests\Rector\MethodCall\SimplifyFormRenderingRector\SimplifyFormRenderingRectorTest
+ */
+final class SimplifyFormRenderingRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Symplify form rendering by not calling `->createView()` on `render` function',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+class ReplaceFormCreateViewFunctionCall extends AbstractController
+{
+    public function form(): Response
+    {
+        return $this->render('form.html.twig', [
+            'form' => $form->createView(),
+        ]);
+    }
+}
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+class ReplaceFormCreateViewFunctionCall extends AbstractController
+{
+    public function form(): Response
+    {
+        return $this->render('form.html.twig', [
+            'form' => $form,
+        ]);
+    }
+}
+CODE_SAMPLE
+                ),
+            ]
+        );
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isObjectType(
+            $node->var,
+            new ObjectType('Symfony\Bundle\FrameworkBundle\Controller\AbstractController')
+        )) {
+            return null;
+        }
+
```

Comment:
first class callable should be skipped:

```suggestion
        if ($node->isFirstClassCallable()) {
            return null;
        }
```

Reply:
++